lib/TunesTakeoutWrapper.rb

```diff
@@ -0,0 +1,50 @@
+require 'httparty'
+
+class TunesTakeoutWrapper
+	BASE_URL = "https://tunes-takeout-api.herokuapp.com/v1"
+
+	def self.get_suggestion(suggestion_id)
+		HTTParty.get(BASE_URL + "/suggestions/" + "#{suggestion_id}").parsed_response["suggestion"]
+	end
+	
+	def self.top_suggestions
+		top_twenty = HTTParty.get(BASE_URL + "/suggestions/top").parsed_response["suggestions"]
+		suggestion_hashes = []
```

Comment:
I would consider renaming this to `suggestions_array` because "hashes" is misleading because it's an array.